### PR TITLE
Update test suites to reflect api changes to the permissions library

### PR DESCRIPTION
**Pull request by saturnial**

### CODE REVIEW

File: test/ts/unit/collateralizer/collateralizer.ts [hunk 1]
```diff
@@ -172,6 +176,61 @@ contract("CollateralizedContract (Unit Tests)", async (ACCOUNTS) => {
...
+            before(async () => {
+                txHash = await collateralizerContract.addAuthorizedCollateralizeAgent.sendTransactionAsync(
+                    AGENT,
+                );
+            });
+
+            it("adds specified agent to set of authorized agents", async () => {
+                const authorizedAgents = await collateralizerContract.getAuthorizedCollateralizeAgents.callAsync();
+                expect(authorizedAgents.includes(AGENT)).to.be.true;
+            });
```
Comment (nadavhollander) [inline test/ts/unit/collateralizer/collateralizer.ts]: Please add a test for ensuring this is not a method that can be called by non-contract owner.

File: test/ts/unit/collateralizer/collateralizer.ts [hunk 2]
```diff
@@ -172,6 +176,61 @@ contract("CollateralizedContract (Unit Tests)", async (ACCOUNTS) => {
...
+            before(async () => {
+                txHash = await collateralizerContract.revokeCollateralizeAuthorization.sendTransactionAsync(
+                    AGENT,
+                );
+            });
+
+            it("removes specified agent from set of authorized agents", async () => {
+                const authorizedAgents = await collateralizerContract.getAuthorizedCollateralizeAgents.callAsync();
+                expect(authorizedAgents.includes(AGENT)).to.be.false;
+            });
```
Comment (nadavhollander) [inline test/ts/unit/collateralizer/collateralizer.ts]: Likewise here.

File: test/ts/unit/debt_token.ts [hunk 3]
```diff
@@ -154,38 +163,168 @@ contract("Debt Token (Unit Tests)", (ACCOUNTS) => {
...
+                        txHash,
+                        EventNames.AuthorizationRevoked,
+                    );
+                    expect(queryResult).to.deep.equal(expectedLogEntry);
+                });
+
+                it("should not list agent as authorized", async () => {
+                    const authorizedAgents = await debtToken.getAuthorizedMintAgents.callAsync();
+                    await expect(authorizedAgents.includes(AUTHORIZED_MINT_AGENT)).to.be.false;
+                });
```
Comment (nadavhollander) [inline test/ts/unit/debt_token.ts]: Please add test for making sure non-owner cannot revokeMintAgentAuthorization

File: test/ts/unit/debt_token.ts [hunk 4]
```diff
@@ -154,38 +163,168 @@ contract("Debt Token (Unit Tests)", (ACCOUNTS) => {
...
+                        debtToken.address,
+                        AUTHORIZED_URI_AGENT,
+                        "debt-token-uri",
+                    );
+                    const queryResult = await queryLogsForEvent(
+                        txHash,
+                        EventNames.AuthorizationRevoked,
+                    );
+                    expect(queryResult).to.deep.equal(expectedLogEntry);
+                });
```
Comment (nadavhollander) [inline test/ts/unit/debt_token.ts]: Please add test for making sure non-owner cannot revoke URI authorization

**Comment by saturnial** [comment]
♻️ @nadavhollander

**Pull request merged**